core-api/src/main/java/org/silverpeas/core/persistence/jdbc/sql/JdbcSqlQuery.java

```diff
@@ -476,8 +575,8 @@ public long execute() throws SQLException {
    * @throws java.sql.SQLException on SQL error.
    */
   public long executeWith(Connection connection) throws SQLException {
-    String builtSqlQuery = getSqlQuery();
-    if (builtSqlQuery.startsWith("select count(*)")) {
+    String builtSqlQuery = getSqlQuery().trim();
+    if ("SELECT COUNT(".equalsIgnoreCase(builtSqlQuery.substring(0, 13))) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Assign this magic number 13 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS109)